cashctrl_api/client.py

```diff
@@ -375,12 +343,48 @@ def list_files(self) -> pd.DataFrame:
         """
         files = pd.DataFrame(self.get("file/list.json")['data'])
         columns_except_path = {key: value for key, value
-                               in self.FILE_COLUMNS.items() if key != 'path'}
+                               in FILE_COLUMNS.items() if key != 'path'}
         df = enforce_dtypes(files, columns_except_path)
         if len(df) > 0:
             categories = self.list_categories('file')[['path', 'id']]
             categories = categories.rename(columns={'id': 'categoryId'})
             df = df.merge(categories, on='categoryId', how='left')
             df['path'] = df['path'].fillna('') + '/' + df['name']
-        df = enforce_dtypes(df, self.FILE_COLUMNS)
+        df = enforce_dtypes(df, FILE_COLUMNS)
         return df.sort_values('path')
+    
+    def list_tax_rates(self) -> pd.DataFrame:
+        """
+        List remote tax rates with their attributes.
+
+        Returns:
+            pd.DataFrame: A DataFrame with CashCtrlClient.TAX_COLUMNS schema.
+        """
+        tax_rates = pd.DataFrame(self.get("tax/list.json")['data'])
+        df = enforce_dtypes(tax_rates, TAX_COLUMNS)
+
+        return df.sort_values('value')
+    
+    def list_accounts(self) -> pd.DataFrame:
+        """
+        List remote accounts with their attributes.
+
+        Returns:
+            pd.DataFrame: A DataFrame with CashCtrlClient.ACCOUNT_COLUMNS schema.
+        """
+        accounts = pd.DataFrame(self.get("account/list.json")['data'])
+        df = enforce_dtypes(accounts, ACCOUNT_COLUMNS)
```

Comment:
Similar to files, accounts in CashCtrl have (nested) categories attached.
We should also return the category in unix-style path notation (e.g. '/Assets/Current Assets/Liquid Funds'). See `list_files()` for an example.

Reply:
fixed